### ISSUE TITLE
Fundamental Data

### ISSUE THREAD

**Issue by quaid281** [hydrosquall/tiingo-python#457]
Is there documentation for how to get fundamental data yet?

**Comment by jafetgado** [comment]
Yes, I am mostly interested in fundamental data, not price. Is there a way to use to do with Python/API?

**Comment by hydrosquall** [comment]
Hi,

We haven't added it to this library yet since the endpoint was in beta, but we look into doing so.

In the meantime, there is documentation for the raw HTTP endpoints that can be used in the meantime:

https://api.tiingo.com/documentation/fundamentals

**Comment by hydrosquall** [comment]
This is being taken care of in these two:

- https://github.com/hydrosquall/tiingo-python/pull/581
- https://github.com/hydrosquall/tiingo-python/pull/596
